data/remnant missions.txt

```diff
@@ -495,3 +560,759 @@ mission "Remnant: Technology Available"
+			`	Any salvage or unrecognized technology gets scanned in our quarantine area for threats, then looked over to figure out if there is anything new in it. If there is something new, it is shipped to a lab for our researchers to dissect. Anything we have already figured out ends up in here, where Remnant captains are welcome to use it as they see fit."`
+			choice
+				`	"Threats?"`
+					goto threats
+				`	"Interesting."`
+			`	"Yes, it is, isn't it? So many new things out there to discover and learn about.`
+				goto end
+			label threats
+			`	"Yes, sometimes salvage that is brought in is dangerous. System Cores come with fleets of micro-bots, for instance. If they aren't de-activated properly, they keep trying to rebuild their surroundings in the form of the last ship they were configured to serve. And while it hasn't happened yet, we are worried that someday we might run into someone who builds traps into their equipment, or uses nanobots that could run amok. We haven't seen any of that yet, but our science-fiction stories are filled with tales of nanotech apocalypses and things like that.`
+			label end
+			`	Well, I need to get back to work. That last battle put us behind schedule on our repairs. Have a good day!" With a final flourish that you think is the equivalent of an exclamation mark, Tali strides out of the outfitters, heading towards another landing bay.`
+
+event "Remnant Salvage Available"
+	outfitter "remnant"
+		"Generator (Furnace Class)"
+		"Generator (Candle Class)"
+		"Triple Plasma Core"
+		"Small Heat Shunt"
+		"Large Heat Shunt"
+		"Systems Core (Small)"
+		"Thruster (Asteroid Class)"
+		"Thruster (Planetary Class)"
+		"Steering (Asteroid Class)"
+		"Steering (Planetary Class)"
+		"Thruster (Comet Class)"
+		"Steering (Comet Class)"
+		"Korath Fire-Lance"
+	
+
+mission "Remnant: Expanded Horizons Quarg 1"
+	name "Visiting the Quarg"
+	description "Take Remnant researchers to see some Quarg."
+	minor
+	passengers 3
+	blocked "This mission requires bunks for three passengers."
+	source
+		government "Remnant"
+	destination "Wayfarer"
+	to offer
+		has "Remnant: Learning Sign 2: done"
+		has "First Contact: Quarg: offered"
+		random < 40
+	on offer
+		conversation
+			`As you relax in the cafeteria on <planet> you are approached by a young Remnant. Judging by the hand-held sensors and tools hanging off her belt, you suppose she is a researcher or an engineer.`
+			`	"Hello Captain <last>. My name is Dawn, and I have been spending the past few days skimming the archive you retrieved. I wanted to ask you a few questions. May I?"`
```

Comment:
The archives I retrieved? This mission's `to offer` doesn't check to see if I even did that. :)

Reply:
fixed. Thanks!